README.md

```diff
@@ -1,52 +1,147 @@
-## BEGIN - TO DELETE TILL END
+# reshuffle-jira-connector
 
-THIS IS A TEMPLATE REPO FOR NEW RESHUFFLE CONNECTORS
-1. Create a new connector repo from this template using this link https://github.com/reshufflehq/reshuffle-template-connector/generate
-2. Clone the repo locally
-3. Rename all occurrences of _CONNECTOR_NAME_
-4. `npm install`
-5. `npm run build:watch`
-6. Implement your events/actions in `src/index.ts`
-7. `npm run lint`
-8. Push your code
-9. Go to https://app.circleci.com/projects/project-dashboard/github/reshufflehq/
-    a. You should see your new connector repo
-    b. click on `Set Up Project` for the repo
-    c. click on `Use Existing Config`
-    d. click on `Start Building`
+[Code](https://github.com/reshufflehq/reshuffle-jira-connector) |  [npm](https://www.npmjs.com/package/reshuffle-jira-connector) | [Code sample](https://github.com/reshufflehq/reshuffle/tree/master/examples/jira)
 
-10. If circle CI checks are all green, you are all set!
+`npm install reshuffle-jira-connector`
 
-// Keep documentation template below
+This connector uses [Node Jira Client](https://github.com/jira-node/node-jira-client) package.
 
-## END
+### Reshuffle Jira Connector
 
-# reshuffle-_CONNECTOR_NAME_-connector
+This connector provides a connector for [Atlassian Jira](https://www.atlassian.com/software/jira).
 
```

Comment:
As @ashevat suggested, it is a good idea to have a fully working example at the top of the README

Reply:
We have a link to the example (code sample) at the top of the Readme but you can't directly see the example.
<img width="429" alt="Screen Shot 2020-11-12 at 9 23 41 AM" src="https://user-images.githubusercontent.com/16469234/98860591-cc06f800-24c8-11eb-94e3-cc00a0fe0368.png">

I'll add one.